strongbox-commons/src/main/java/org/carlspring/strongbox/util/ThrowingFunction.java

```diff
@@ -10,7 +10,7 @@
 {
     R apply(T t) throws E;
 
-    static <T, R, E extends Throwable> Function<T, R> unchecked(ThrowingFunction<T, R, E> function)
+    static <T, R, E> Function<T, R> unchecked(ThrowingFunction<T, R, Throwable> function)
```

Comment:
@sbespalov :

Are you happy with this change?

Reply:
I'm fine if it compiles well :)  also `E` seems don't needed at `<T, R, E>`.

Reply:
Yeah, I was wondering, if I shouldn't look into this as well. Thanks!

Reply:
Well, it seems that doesn't work either. I'll leave it as it is for now.

If somebody feels stronly about it and knows how to fix it, that'll be great! For now, it compiles and works.